### DISCUSSION TITLE
Why is HOST/PORT not included in `forceRuntimeEnvVars` by default?

### DISCUSSION THREAD

**Discussion by squiddy** [jaredpalmer/razzle#1606] [General] [answered]
By default HOST and PORT environment variables are embedded in the build. That means unless one changes the configuration, changing the port a server runs on later is not possible.

It's not a problem, changing the configuration is easy and some of the examples even do that. I'm just wondering if there is a use-case I'm missing and why it couldn't include those keys by default.

**Comment by fivethreeo** [comment, marked as answer]
It is mostly to work as create-react-app does by default. So you can use PORT and HOST in client js without passing it from the server js.
